scripts/assets.sh

```diff
@@ -6,7 +6,7 @@ set -e -x
 
 yarn clean
 
-NODE_ENV=production node_modules/.bin/webpack
+NODE_ENV=production yarn webpack
 stylus \
```

Comment:
@damassi wanted to check -- now that we are using the assets compiled by webpack, is the stylus compression here necessary?

Reply:
The way it works in force is that when we started using the asset manifest plugin (which was a PR _after_ the webpack refactor) we build and [seed the CSS here](https://github.com/artsy/force/blob/master/webpack/envs/productionConfig.js#L29). 

I think removing this is for a later PR, once bucket-assets is ready to be removed completely along with its asset hashing.